### DISCUSSION TITLE
binary function in python

### DISCUSSION THREAD

**Discussion by connuna** [enzifiri/GitHub-Achievements#731] [Q&A] [answered]
How can I create a function that enters numbers and converts it to binary in python

**Comment by madmarin** [comment, marked as answer]
Hello!

To convert a decimal number to binary in Python, you can use the built-in `bin()` function, which returns the binary representation of a number as a string prefixed with "0b". Here's a simple example:

```python
def decimal_to_binary(n):
    return bin(n).replace("0b", "")

# Example usage:
decimal_number = 10
binary_number = decimal_to_binary(decimal_number)
print(f"The binary representation of {decimal_number} is {binary_number}")

**Reply by connuna** [reply]
thank youuu so much brooo
